x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/parser/IdentifierBuilder.java

```diff
@@ -138,4 +147,20 @@ private static void validateIndexPattern(String indexPattern, EsqlBaseParser.Ind
     private static String removeExclusion(String indexPattern) {
         return indexPattern.charAt(0) == EXCLUSION.charAt(0) ? indexPattern.substring(1) : indexPattern;
     }
+
+    private static boolean patternExceedsMaxIndexSeparators(String pattern, int maxAllowedSeparators) {
+        int seperatorsCount = 0;
+        boolean inDateTime = false;
+        for (char ch : pattern.toCharArray()) {
+            if (ch == '<') {
```

Comment:
Regarding this check for `<` and `>` (since I don't know all the fine details of ES validation), can you point us to what you mentioned in the description of the PR - our spec around these restrictions?

> more than 1 index seperator (not valid as per our current spec)